code/modules/uplink/uplink_items.dm

```diff
@@ -1632,6 +1632,11 @@ datum/uplink_item/stealthy_tools/taeclowndo_shoes
 	item = /obj/item/aiModule/syndicate
 	cost = 9
 
+/datum/uplink_item/device_tools/grusimov_module
+	name = "Grusimov AI Law Upload Module"
+	desc = "When used with an upload console, this module allows you to upload priority laws to an artificial intelligence."
+	item = /obj/item/aiModule/core/full/grusimov
+	cost = 12
```

Comment:
complete lawsets don't belong in uplinks and it costing 12 tc is ridiculous

Reply:
tc cost can change, but it's part of a larger thing, which many players and community members came up with. This can certainly be put to vote.

Reply:
It's a part of larger thing that is not added in this PR. Just make the change when you will add "rest".